usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -44,6 +44,7 @@ The currently available endpoints are listed in the following table.
 |[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/program_activity/count/](/api/v2/agency/012/program_activity/count/)|GET| Returns the count of Program Activity categories for the agency in a single fiscal year |
 |[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/sub_agency/](/api/v2/agency/012/sub_agency/)|GET| Returns a list of sub-agencies and offices with obligated amounts, transaction counts and new award counts for the agency in a single fiscal year |
 |[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/sub_agency/count/](/api/v2/agency/012/sub_agency/count/)|GET| Returns the number of sub-agencies and offices for the agency in a single fiscal year |
+|[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/subcomponents/](/api/v2/agency/012/subcomponents/)|GET| Returns a list of bureaus for the agency in a single fiscal year |
```

Comment:
Great question, the tickets both say `subcomponents` which is what i was looking at when making the url, but it's unclear. I'll change it to match the contract since I think that makes the most sense at this point